libs/common/src/sql-db-connection/migrations/1709731032407-AddOnDeleteCascadeToAzureFileInfo.ts

```diff
@@ -0,0 +1,137 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddOnDeleteCascadeToAzureFileInfo1709731032407 implements MigrationInterface {
+  name = 'AddOnDeleteCascadeToAzureFileInfo1709731032407';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            ALTER TABLE "AzureFileInfo" DROP CONSTRAINT "FK_4370b01aed42cea044ee2b1ce6f"
+        `);
+    await queryRunner.query(`
```

Comment:
Why are all these extra fields getting changed? have we changed their types somewhere?

Reply:
For future readers, see [here](https://github.com/UK-Export-Finance/dtfs2/pull/2807/files#r1514714219)